### PR TITLE
Added Kodi icon

### PR DESCRIPTION
**Pull request by JoaoPedroMoraes**
**Issue:** #1490 

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
From the official SVG I was able to create a monocolor version. https://kodi.tv/

### CODE REVIEW

**Comment by davidklebanoff** [comment]
Where did you get the color from. I pulled `#17B2E7` from the header SVG. Perhaps the color in the commit is just a typo?

**Comment by JoaoPedroMoraes** [comment]
> De onde você tirou a cor? Eu puxei do cabeçalho SVG. Talvez a cor no commit seja apenas um erro de digitação?`#17B2E7`

I took this color from that link https://kodi.wiki/view/Official:Media_center_logos

**Comment by davidklebanoff** [comment]
Thanks for the clarification. Since those icons are PNG, I updated the color to match the color embedded in the SVG used on the website's header logo - since it should be more accurate.

**Comment by davidklebanoff** [comment]
Thanks for taking this one on @JoaoPedroMoraes!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1500,6 +1500,11 @@
             "hex": "333333",
             "source": "https://withknown.com/img/logo_k.png"
         },
+        {
+            "title": "Kodi",
+            "hex": "17B2E7",
+            "source": "https://kodi.tv/ "
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Minor issue here, the fact that the source has a trailing space might potentially cause problems for anyone using the source in the [NPM package](https://www.npmjs.com/package/simple-icons). Ideally we would fix this before releasing a new version.

Comment (davidklebanoff) [reply _data/simple-icons.json]: I'll clean that in the develop branch. I'll probably update the linter as well to catch such things in the future.

Comment (davidklebanoff) [reply _data/simple-icons.json]: I'll clean that in the develop branch. I'll look into updating the linter as well to catch such things in the future.

Comment (ericcornelissen) [reply _data/simple-icons.json]: I was already planning to do that as well, thanks for pushing the fix though 👍 See #1502

**Pull request merged**